torch_xla/csrc/helpers.cpp

```diff
@@ -72,6 +95,152 @@ xla::XlaComputation CreateMinMaxComputation(const std::string& name,
   return min_max_computation;
 }
 
+// Joins the given `span` into a `std::string`.
+//
+// For each element, this function will append to the output string:
+//   1. A star (*), if `needs_star(el)` returns true
+//   2. The return value of `get_printable_data(el)`
+template <class T, class FStar, class FPrint = T (*)(T)>
+std::string SpanToStringWithStar(
+    absl::Span<const T> span, FStar&& needs_star,
```

Comment:
Per https://google.github.io/styleguide/cppguide.html#Rvalue_references, avoid Rvalue references here and below.